magento/magento2
# escape logo alt text and title

### Description (*)

Added escapeHtmlattr to logo alt text so theres not XSS possibility by changing this text from admin panel
### Fixed Issues (if relevant)

1. magento/magento2#19269: Logo "alt" attribute not escaped; possible (self) XSS

### Manual testing scenarios (*)


1. Go to Content -> Design -> Configuration
2. Edit a theme (doesn't matter which) by clicking "Edit"
3. Scroll down to "Header" and add the following to the Logo Image Alt field
` " onload="javascript:alert('hello, world');`
4. Check whether you get javascript alert window or not

`Notice the leading "
### Contribution checklist (*)
